.nvmrc

```diff
@@ -0,0 +1 @@
+6.4.0
```

Comment:
The other repos I've seen here seem to be on Node 4. I don't know if it's important to stay on that version or if we can go to 6.